starknet_py/utils/crypto/facade.py

```diff
@@ -44,7 +44,7 @@ class Call:
 @dataclass(frozen=True)
 class MultiCall:
     account: int
-    calls: Call
+    calls: List[Call]
```

Comment:
```suggestion
    calls: Iterable[Call]
```
We probably can generalise it to